packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4419,6 +4489,8 @@ describe('plugin-meetings', () => {
           sipUrl: 'some_sip_url', // or sipMeetingUri
           meetingNumber: '123456', // this.config.experimental.enableUnifiedMeetings
           hostId: 'some_host_id', // this.owner;
+          permissionToken:
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJtb2NrUGFzc3dvcmQiOiJ0aGlzSXNNb2NrUGFzc3dvcmQifQ.3-WXiR8vhGUH3VXO0DTpsTwnnkVQ3vhGQcktwIarj3I',
```

Comment:
do we need a valid looking access token? We could just say `accessToken` right?

Reply:
We need a valid looking access token because previously we were just stubbing the method. Now since we have changed the package it is not so straight forward to stub it. Also if we dont pass a valid access token it fails the test. This permission token has been taken from another test which was using it.